.pipelines/OneBranch.PullRequest.yml

```diff
@@ -86,4 +86,20 @@ extends:
             python -m pip install cibuildwheel
             python -m cibuildwheel --platform windows --output-dir $(REPOROOT)\out
           displayName: Build wheels
-        
\ No newline at end of file
+        - script: |
+            7z x $(REPOROOT)\out\*.whl -o$(REPOROOT)\out\*
+          displayName: unzip the package
+        - task: onebranch.pipeline.signing@1
+          inputs:
+            command: 'sign'
+            signing_profile: 'external_distribution'
+            files_to_sign: '**/**/*.pyd;**/**/*.dll;**/**/*.so'
```

Comment:
*.so is not signable. Please do not include *.so files.

Reply:
*.so files removed